fabric/patches/9_0001-add-proxy-support-for-docker-in-peer.patch

```diff
@@ -1,7 +1,7 @@
-From ffabe25186d43af74bf8dd351d5237865cadc68c Mon Sep 17 00:00:00 2001
+From 2c0c61f6c35ef76d6425eb680bb8173a59453d40 Mon Sep 17 00:00:00 2001
 From: Michael Steiner <michael.steiner@intel.com>
 Date: Wed, 18 Mar 2020 19:42:55 -0700
-Subject: [PATCH] add proxy support for docker in peer
+Subject: [PATCH 1/1] add proxy support for docker in peer
```

Comment:
This patch could probably be reviewed and get in on it's own merits, if there's no alternative provided by upstream.

Reply:
yes, i have that also in the back of my mind. However, i don't think it is really yet a clean and complete patch, just works for us and once all chaincode would migrate to external builder it also would become obsolete.  That said, i guess i could create a PR for it just to gauge if there is interest and also for the record if somebody else is ever running into the issue and so would have a starting point ....